dev/deps/spark-deps-hadoop-2-hive-2.3

```diff
@@ -38,6 +38,7 @@ commons-beanutils/1.9.4//commons-beanutils-1.9.4.jar
 commons-cli/1.5.0//commons-cli-1.5.0.jar
 commons-codec/1.15//commons-codec-1.15.jar
 commons-collections/3.2.2//commons-collections-3.2.2.jar
+commons-collections4/4.4//commons-collections4-4.4.jar
```

Comment:
`hadoop-2.7` profile still need `commons-collections 3.2.2` due to It is the compilation dependency of `hadoop-2.7`.